Resources/Prototypes/_Goobstation/Recipes/Reactions/medicine.yml

```
@@ -11,3 +11,372 @@
+  id: StypticCrystalCreation
+  quantized: true
+  conserveEnergy: false
+  reactants:
+    StypticCrystalCatalyst:
+      amount: 5
+    Blood:
+      amount: 10
+    Sodium:
+      amount: 10
+  effects:
+    - !type:CreateEntityReactionEffect
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)